bpf/node_config.h

```diff
@@ -225,6 +225,7 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 # define WORLD_CIDRS4_MAP test_cilium_world_cidrs4
 # define WORLD_CIDRS4_MAP_SIZE 16384
 #endif /* ENABLE_HIGH_SCALE_IPCACHE */
+#define MCAST_GROUP_OUTER_V4_MAP test_cilium_mcast_group_v4_outer
```

Comment:
I haven't been able to figure out exactly why we alias all these map names, but it looks like all they have in common is being pinned maps. We're planning to get rid of these with the removal of the ELF rewriter over the course of 1.16, and the verifier tests have been running without pinned maps for the course of the 1.14 and 1.15 dev cycles.

Would you be willing to try if anything in CI fails when using a regular map handle (as opposed to a macro) in BPF C code? Or did you already hit some breakage that led you to adding this macro?

Reply:
I guess the aliasing allows the map name to be pushed down from Agent? Isn't this something we may want control of ?

Reply:
> Isn't this something we may want control of ?

In practice, we only do this for per-endpoint maps like cilium_calls, certainly not for global maps! All map renaming behaviour will be replaced by per-endpoint/device bpffs dirs in 1.16, so no need to carry this over into new features.

Reply:
Okay cool, i'll try this removal.

Reply:
This is resolved. I removed the aliasing completely.